randovania/layout/game_patches_serializer.py

```diff
@@ -27,7 +27,7 @@
     from randovania.game_description.pickup.pickup_entry import PickupEntry
     from randovania.layout.base.base_configuration import BaseConfiguration
 
-_ETM_NAME = "Energy Transfer Module"
+_ETM_NAME = "Nothing"
 
```

Comment:
Please renane the variable (and its references) now that its no longer referencing an etm